lib/cog/adapters/slack/rtm_connector.ex

```diff
@@ -112,6 +114,17 @@ defmodule Cog.Adapters.Slack.RTMConnector do
     end
     handler.()
   end
+  def handle_message(%{type: channel_event, channel: channel}, _slack, state) when channel_event in @channel_events do
+    channel_id = case channel do
+      channel when is_binary(channel) ->
+        channel
+      %{id: id} ->
```

Comment:
What determines if `channel` is a binary or a map? Whether it's a "channel" event or a "group" event? If so, a comment might be useful here, just for reference.